src/plugins/music-xml-viewer/music-xml-viewer.yml

```diff
@@ -0,0 +1,9 @@
+name:
+  en: MusicXML viewer
+  de: MusicXML-Anzeige
+zoom:
+  en: Zoom
+  de: Zoom
+caption:
+  en: Caption
+  de: Legende
```

Comment:
Tiny thing: the `pdf-viewer` plugin also has an identical "caption" translation, we could move it to `common.yml`

Reply:
Done!